developer/src/kmcmplib/tests/get_test_source.bat

```diff
@@ -1,2 +1,2 @@
 @echo off
-dir /s/b ..\..\..\..\keyboards\release\*.kmn
+dir /s/b %KEYMAN_ROOT%\..\keyboards\release\*.kmn
```

Comment:
This doesn't look like. Don't we have the `*.kmn` files for testing in the keyman repo?

Reply:
we have ~700 kmn-files for kmcompxtest in keyboards\release\   (e.g. C:\Projects\keyman\keyboards\release\u\ukwuani\source\ukwuani.kmn)
We don`t have these tests in a release-folder  in C:\Projects\keyman\keyman\common\test\keyboards

Reply:
ah, ok!